grafana/provider_test.go

```diff
@@ -85,7 +85,7 @@ func TestProviderConfigure(t *testing.T) {
 		{
 			name:        "no config",
 			env:         map[string]string{},
-			expectedErr: "\"auth\": one of `auth,cloud_api_key,sm_access_token` must be specified",
+			expectedErr: "\"auth\": one of `auth,cloud_api_key,oncall_access_token,sm_access_token` must\nbe specified",
```

Comment:
I'm curious: where does this `\n` between `must` and `be` come from? Checking the [SDK code](https://github.com/hashicorp/terraform-plugin-sdk/blob/4681738a561387fb0b3aaa69aeb42231383634a0/helper/schema/schema.go#L1874) that raises this error doesn't seem to have a newline, maybe it's the `resource.UnitTest`? 🤔